pkg/recipes/driver/terraform.go

```diff
@@ -174,9 +177,13 @@ func (d *terraformDriver) prepareRecipeResponse(ctx context.Context, definition
 		TemplateVersion: definition.TemplateVersion,
 	}
 
-	deployedResources, err := d.getDeployedOutputResources(ctx, tfState.Values.RootModule)
-	if err != nil {
-		return &recipes.RecipeOutput{}, err
+	var deployedResources []string
+	if tfState.Values != nil && tfState.Values.RootModule != nil {
```

Comment:
Same question here as above: Could you look at this part and let me know if this makes sense @kachawla @vishwahiremat? Can the **Values** ever be nil?